lib/activerecord/tablefree.rb

```diff
@@ -78,7 +78,7 @@ def load_schema!
         @columns_hash.each do |name, column|
           define_attribute(
             name,
-            connection.lookup_cast_type_from_column(column),
```

Comment:
I'm curious what `connection.lookup_cast_type_from_column` was meant to do when `column.sql_type_metadata` does the job...

Reply:
I can't remember, but this was originally written for Rails 2, then rewritten for Rails 3 & 4, then again for Rails 5, and some legacy stuff was certainly carried over from the before times, when Rails didn't have such niceties. :). Thanks!